README.md

```diff
@@ -1,2 +1,61 @@
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+<a href="mailto:sales@syncfusion.com?Subject=Syncfusion Vue Grid - GitHub" target="_top">sales@syncfusion.com</a> | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=vue-chart-github-samples)">www.syncfusion.com</a> | Toll Free: 1-888-9 DOTNET <br>
+</p>
```

Comment:
![image](https://github.com/SyncfusionExamples/getting-started-with-the-vue-charts-component/assets/95272441/32d9a504-371c-4d6b-b44d-8dcf98d0c4b4)

Reply:
Removed